src/org/omegat/gui/preferences/view/TMMatchesPreferencesController.java

```diff
@@ -88,6 +88,9 @@ protected void initFromPrefs() {
                 .setSelectedItem(Preferences.getPreferenceEnumDefault(Preferences.EXT_TMX_SORT_KEY, SORT_KEY.SCORE));
         panel.displayLevel2Tags.setSelected(Preferences.isPreference(Preferences.EXT_TMX_SHOW_LEVEL2));
         panel.useSlash.setSelected(Preferences.isPreference(Preferences.EXT_TMX_USE_SLASH));
+        panel.keepForeignMatches.setSelected(Preferences.isPreference(Preferences.EXT_TMX_KEEP_FOREIGN_MATCH));
```

Comment:
I think it would be nice if toggling the "Include matches from other languages" checkbox enabled/disabled the "Penalty for matches in other languages" label and spinner. That would reinforce the relationship between the two.

Reply:
Good idea. I added this behavior.